lib/routes/home/widgets/payments_list/payment_item_title.dart

```diff
@@ -22,8 +22,35 @@ class PaymentItemTitle extends StatelessWidget {
   }
 
   String _title(BuildContext context) {
-    final description = _paymentInfo.description;
-    if (description != null && description.isNotEmpty) return description;   
+    final description = _paymentInfo.description?.replaceAll("\n", " ").trim();
+    if (description != null && description.isNotEmpty) {
+      final breezPosRegex = RegExp(r'(?<=\|)(.*)(?=\|)');
```

Comment:
What is this regular expression matching?

Reply:
Breez POS payment/sale, e.g.:

```
id                                                                payment_type  payment_time  amount_msat  fee_msat  pending  description                                                                                                                                                                                                           details
----------------------------------------------------------------  ------------  ------------  -----------  --------  -------  --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------  ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
d7a511c83e32c5b0f49a5a6f5954c7d37408eeee21990a00394256ef40b5cdce  Sent          1672839494    1000         -2000     0         |
João Linus | https://storage.googleapis.com/download/storage/v1/b/breez-technology.appspot.com/o/1216%2F7aaf%2F20901c3890fe729e7c6ca30007d770d3760a4829ae9662e60fa97a0b.png?generation=1658427826135832&alt=media  {"payment_hash":"d7a511c83e32c5b0f49a5a6f5954c7d37408eeee21990a00394256ef40b5cdce","label":"","destination_pubkey":"036af19a240070280278d0a613e7b717bd33dd0f848dc326dfaea2881565bb9d0b","payment_preimage":"","keysend":false,"bolt11":"lnbc10n1p3mtqfkpp567j3rjp7xtzmpay6tfh4j4x86d6q3mhwyxvs5qpegftw7s94eh8qd24yp7q5jn0cw3k7gzvd9h82ueq0ssxsar5wpen5te0wd6x7unpvajjuem0dankcetpwp5hxtnrdakj7er0wahxcmmpvshhxar0wfskwef0wccj7c30vfex2et6946x2cmgdehkcmm80yhxzurswdcx7apwvdhk6tm09ucnyvfky5eyvdmpv9nz2vjxxgcrjvp3vvenswfsvejnwv3ev5mkxdnrvyenqvpsxajrwdesvsenwd3svy6rsv3ev9jnjd3kxfjnvvrxvyunwcfsvgh8qmn88ank2mn9wfshg6t0dc7nzd348q6rydecxgmrzve48qenyfnpd36r6mt9v35kzcqzpgxqrrssrzjqvgptfurj3528snx6e3dtwepafxw5fpzdymw9pj20jj09sunnqmwpapyqqqqqqqql5qqqqlgqqqqqqgq9qsp58a03dy5tat7hc8phac3z8tlu93ertmdgaqkjrq6tfzmnkpqfte7s9qyyssq2j2e5j2h9sd4hn0hdpq5auh94wqtz4wnjnay5w53p5r2y4tqpj2zgfxufqjx6ylgmzg6j2zu2g2hlepg0nch2mx4svuh655w6yg4e4gpw2ueg9"}
```